sources/packages/api/src/route-controllers/education-program/models/summary-education-program.dto.ts

```diff
@@ -14,9 +15,11 @@ export interface SubsetEducationProgramDto {
   name: string;
   description: string;
   credentialType: string;
+  credentialTypeToDisplay: string;
   cipCode: string;
   nocCode: string;
   sabcCode: string;
   approvalStatus: string;
   programIntensity: ProgramIntensity;
+  institutionProgramCode?: string;
```

Comment:
Please remove the ? for the institutionProgramCode, as this DTO is used only for the GET request.

Reply:
I am not following. Why DTO for a GET request is a justification to not use the nullable "?"?
If the property is nullable it could be better expressed by making it explicit. Does it make sense?

Reply:
I am pretty sure, I got a similar comment from @andrewsignori-aot in my previous PRs, so thought of suggesting :) . Its okay, if people are following any one type of DTO creation for GET.

Reply:
@guru-aot, I am pretty sure that we are not on the same page. I would be really, really interested in seeing this comment from any last PR.

Reply:
We talked, thanks for the explanation @andrewsignori-aot